src/ethereum/ethereumTypes.ts

```diff
@@ -260,7 +260,7 @@ export interface EthereumTxOtherParams {
 export const asEthereumTxOtherParams = asObject<EthereumTxOtherParams>({
   from: asArray(asString),
   to: asArray(asString),
-  gas: asString,
+  gasLimit: asString,
```

Comment:
You'll need to update txRpcParamsToSpendInfo since the spec is to use 'gas'.

Reply:
I'll revert to avoid potential regressions here. Given the level of how many ts-expect-errors may be present in this repo, this change is too risky.